build-system/tasks/check-links.js

```diff
@@ -72,9 +73,10 @@ function checkLinks() {
     });
     if (deadLinksFound) {
         util.log(
-            util.colors.red('ERROR'), 'Dead links found. Please update',
-            util.colors.magenta(filesWithDeadLinks.join(',')));
-       process.exit(1);
+            util.colors.yellow('WARNING'),
+            'Dead links potentially found. Please update',
+            util.colors.magenta(filesWithDeadLinks.join(',')),
+            'if necessary.');
```

Comment:
Correct, this is for the case where many md files are edited in one PR, so all dead links can be detected in one go. Saves the trouble of having to re-test after every fix.